api/Confidence_public_api.json

```diff
@@ -106,33 +106,6 @@
       }
     ]
   },
-  {
-    "className": "ConfidenceClientOptions",
-    "apiFunctions": [
-      {
-        "name": "init(credentials:region:initializationStrategy:)",
-        "declaration": "public init(\n    credentials: ConfidenceClientCredentials,\n    region: ConfidenceRegion? = nil,\n    initializationStrategy: InitializationStrategy = .fetchAndActivate\n)"
-      }
-    ]
-  },
-  {
-    "className": "ConfidenceClientCredentials",
-    "apiFunctions": [
-      {
-        "name": "getSecret()",
-        "declaration": "public func getSecret() -> String"
-      }
-    ]
-  },
-  {
-    "className": "ConfidenceMetadata",
-    "apiFunctions": [
-      {
-        "name": "init(name:version:)",
-        "declaration": "public init(name: String, version: String)"
-      }
-    ]
-  },
   {
     "className": "Event",
```

Comment:
It's a bummer that this (IMO internal) class leaks through the EventProducers.

Reply:
I believe in android this Event is another type of Event and the `EngineEvent` is another type.